src/components/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { ButtonProps } from '../Button';
+import style from './IconButton.st.css';
+import { ButtonNext } from 'wix-ui-core/button-next';
+
+export interface IconButtonProps extends ButtonProps {
+  icon: React.ReactNode;
+  isFull?: boolean;
```

Comment:
I'm sorry for driving you crazy with this 😬 
But i'm still not sure how we should add the `full` prop.
Now i'm thinking maybe we should have a `skin` prop with: 
```
enum Skins {
  Line = 'line';
  Full = 'full';
}
```
and `Line` will be default.
This way, if in the future we want to add another skin, we can add it here.
What do you think?